packages/website/package.json

```diff
@@ -27,8 +27,8 @@
     "magic-sdk": "^6.2.1",
     "multiformats": "^9.6.4",
     "next-mdx-remote": "^4.0.0-rc.1",
-    "nextra": "^2.0.0-beta.5",
-    "nextra-theme-docs": "^2.0.0-beta.5",
+    "nextra": "2.0.0-beta.5",
+    "nextra-theme-docs": "2.0.0-beta.5",
```

Comment:
can we revert this please

Reply:
There seems to have been a breaking change in nextra 2.0.0-beta.6, which I didn't have time to figure out the details of last week. It will show up on `main` if you delete the lockfile and `yarn install` again. 

I'd rather pin the dependency here and fix the issue in a separate PR, if that works for you.